lesson06/assignment/css/_size.variables.scss

```diff
@@ -0,0 +1,30 @@
+/*rem sizes*/
+$half: 0.5rem;
+$one: 1rem;
+$onehalf: 1.5rem;
+$twohalf: 2.5rem;
+$three: 3rem;
+$four: 4rem;
+$five: 5rem;
+$ten: 10rem;
+/*px to rem sizes*/
+$two: 0.125rem;
+$seven: 0.4375rem;
+$ten: 0.625rem;
+$hundred: 6.25em;
+$thousand: 62.5rem;
+/*percentages*/
+$zeroper: 0%;
+$oneper: 1%;
+$twoper: 2%;
+$threeper: 3%;
+$tenper: 10%;
+$twentyper: 20%;
+$fortyper: 40%;
+$quarterper: 25%;
+$fiftyfiveper: 55%;
+$seventyper: 70%;
+$eightyper: 80%;
+$ninetyper: 90%;
+$hundredper: 100%;
+$fiftyper: 50%;
```

Comment:
While it's a good idea to turn commonly used values into variables, I think this is overkill. In particular, the names being the percentages themselves indicate that this isn't going to reduce complexity in the code. 

One use case for making percentages variables would be if the site had a commonly used layout, say a sidebar and main content, each a standard size like 20% sidebar, 70% main, and both converting to 90% at a given breakpoint. Creating variables like `$standard-layout-sidebar-width` and `$standard-layout-main-width` to hold the 20% and 70% values, and a `$standard-layout-full` for the 90% would be helpful in allowing sitewide design adjustments to be made easily. Changing these variables' values in this single location would change the appearance of all pages using the standard layout, with far fewer opportunities for bugs than having to change them on every page's stylesheet.